api/api-feeds-api-v1/build.gradle

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
```

Comment:
Update copyright year to 2020?

Maybe we change it for all the files?

Reply:
Done in https://github.com/gocd/gocd/pull/7555/commits/175304df230bc03b8c43810bf2b2cc7eb5832c1b